src/set_theory/ordinal_arithmetic.lean

```diff
@@ -1029,6 +1045,16 @@ begin
   exact lt_irrefl _ this
 end
 
+@[simp] theorem lsub_eq_zero_iff {ι} {f : ι → ordinal} : lsub f = 0 ↔ is_empty ι :=
+begin
```

Comment:
Provided I'm understanding, the idea of this is that the lsub of any nonempty family should be non-zero, right? Do you think we should have a lemma stating this, even if only one direction of it: `0 < lsup f` with the instance that `ι` is nonempty?

Reply:
Yes, and @YaelDillies actually recommended this on the follow-up PR #11270. I'll apply some of those suggestions over here.